apps/tng-dictionary/src/components/Home/Home.tsx

```diff
@@ -1,120 +1,127 @@
-                  <SearchIcon />Paradigms & Vocabulary Lists
-                </Button>
-              </motion.div>
-            </Link>
+    const [open, setOpen] = React.useState(true)
+    const handleClose = () => {
+        setOpen(false)
+    }
+    const handleToggle = () => {
+        setOpen(!false)
+    }
+    const backdrop = (
+        <Backdrop
+            sx={{ color: '#fff', zIndex: '100', background: 'rgb(168,4,4, .8)' }}
+            open={open}
+            onClick={handleClose}
+        >
+            <div style={{ padding: '15px' }}>
+                <h1>Disclaimer</h1>
+                <Divider sx={{ bgcolor: 'white' }} />
+                <p>
+                    The 'Tŝilhqot'in Dictionary' is still under active development and this domain
+                    is intended for testing purposes by those selected to give user feedback.
+                </p>
+                <p>Please do not distribute this app's address.</p>
+                <p>Sechanalyagh,</p>
+                <br />
+                <p>
+                    <img
+                        style={{ marginRight: '10px', verticalAlign: 'text-bottom' }}
+                        src="https://www.tsilhqotin.ca/wp-content/uploads/2022/02/imageedit_14_8913908156.png"
+                        height={20}
+                    ></img>
+                    Tŝilhqot'in National Government
+                </p>
+                <Divider sx={{ bgcolor: 'white' }} />
+                <p style={{ textAlign: 'center' }}>'Click anywhere to continue'</p>
+            </div>
+        </Backdrop>
+    )
+    return (
+        <div className="homeScreen">
+            <motion.div
+                initial={{ opacity: 0 }}
+                animate={{ opacity: 1 }}
+                exit={{ opacity: 0 }}
+                transition={{ duration: 0.6 }}
+            >
+                <div className="home">
+                    <div className="Center">
+                        <Link to="/terms">
+                            <motion.div whileHover={{ scale: 1.05 }} whileTap={{ scale: 0.95 }}>
+                                <Button className="button" style={style} variant="outlined">
+                                    <SearchIcon className="icon" />
+                                    Terms
+                                </Button>
+                            </motion.div>
+                        </Link>
 
-            <Link to="/credits">
-              <motion.div
-                whileHover={{ scale: 1.05, }}
-                whileTap={{ scale: 0.95 }}>
-                <Button className='button' style={style} variant='outlined'>
-                  <InfoOutlinedIcon className='icon' />Credits
-                </Button>
-              </motion.div>
-            </Link>
+                        <Link to="/VocabularyLists">
+                            <motion.div whileHover={{ scale: 1.05 }} whileTap={{ scale: 0.95 }}>
```

Comment:
this bit (whileHover / whileTap) is repeated a lot. I'd aim to have a single source of truth for these settings so that you can update their values for all buttons by making one single change.

same might go for style and variant props.